packages/download-runtime/package.json

```diff
@@ -24,7 +24,8 @@
     "@ampproject/toolbox-core": "^2.0.0",
     "@ampproject/toolbox-runtime-version": "^2.0.0",
     "cross-fetch": "3.0.4",
-    "fetch-mock": "9.2.1"
+    "fetch-mock": "9.2.1",
```

Comment:
Please remove. This should already be included as a dependency (in the root package).